src/lizaalert/courses/admin.py

```diff
@@ -278,3 +278,29 @@ class SubscriptionAdmin(admin.ModelAdmin):
     ]
 
     ordering = ("-updated_at",)
+
+
+@admin.register(Division)
+class DivisionAdmin(admin.ModelAdmin):
+    """Aдминка для Division."""
+
+    ordering = ("-updated_at",)
+    list_display = ("title", "author", "updated_at", "courses")
+    exclude = ("author",)
+    list_select_related = ("author",)
+
+    @admin.display(description="Курсы")
+    def courses(self, obj):
+        """Показывает курсы по этим направлениям."""
+        return [[course.title] for course in obj.course_set.all()]
+
+    def get_queryset(self, request):
+        qs = self.model._default_manager.get_queryset().prefetch_related("course_set")
+        ordering = self.get_ordering(request)
+        if ordering:
+            qs = qs.order_by(*ordering)
+        return qs
```

Comment:
используй запрос базового класса

```suggestion
        return super().get_queryset(request).prefetch_related("course_set")
```